app/_src/kubernetes-ingress-controller/guides/high-availability/fallback-config.md

```diff
@@ -0,0 +1,476 @@
+        {"kind": "HTTPRoute", "name": "route-b", "namespace": "default"}
+    ],
+    "excludedObjects": [
+        {"kind": "KongPlugin", "name": "key-auth", "namespace": "default"}, 
+        {"kind": "HTTPRoute", "name": "route-b", "namespace": "default"}
+    ]
+}
+```
+
+#### Verifying the working route is still operational and can be updated
+
+We can also ensure the other `HTTPRoute` is still working:
+
+```bash
+curl -i $PROXY_IP/route-a
+```
+
+The results should look like this:
+```text
+HTTP/1.1 200 OK
+Content-Type: text/plain; charset=utf-8
+Content-Length: 137
+Connection: keep-alive
+X-Kong-Upstream-Latency: 1
+X-Kong-Proxy-Latency: 1
+Via: kong/3.6.0
+X-Kong-Request-Id: 5bf50016730eae43c359c17b41dc8614
+
+Welcome, you are connected to node orbstack.
+Running on Pod echo-74c66b778-szf8f.
+In namespace default.
+With IP address 192.168.194.13.
+```
+
+What's more, we're still able to update the correct `HTTPRoute` without any issues. Let's modify `route-a`'s path:
+
+```bash
+kubectl patch httproute route-a --type merge -p '{"spec":{"rules":[{"matches":[{"path":{"type":"PathPrefix","value":"/route-a-modified"}}],"backendRefs":[{"name":"echo","port":80}]}]}}'
+```
+
+The results should look like this:
+```text
+httproute.gateway.networking.k8s.io/route-a patched
+```
+
+Let's verify the updated `HTTPRoute`:
+
+```bash
+curl -i $PROXY_IP/route-a-modified
+```
+
+The results should look like this:
+```text
+HTTP/1.1 200 OK
+Content-Type: text/plain; charset=utf-8
+Content-Length: 137
+Connection: keep-alive
+X-Kong-Upstream-Latency: 1
+X-Kong-Proxy-Latency: 0
+Via: kong/3.6.0
+X-Kong-Request-Id: f26ce453eeeda50e3d53a26f44f0f21f
+
+Welcome, you are connected to node orbstack.
+Running on Pod echo-74c66b778-szf8f.
+In namespace default.
+With IP address 192.168.194.13.
+```
+
+The Fallback Configuration mechanism has successfully isolated the broken `HTTPRoute` and allowed the correct one to be
+updated.
+
+### Backfilling broken objects
+
+Another mode of operation that the Fallback Configuration feature supports is backfilling broken objects with their last
+valid version. To demonstrate this, we'll use the same setup as in the default mode, but this time we'll enable the
+`--use-last-valid-config-for-fallback` flag.
+
+```bash
+helm upgrade --install kong kong/ingress -n kong \
+--set ingressController.env.feature_gates=FallbackConfiguration=true \
+--set ingressController.env.use_last_valid_config_for_fallback=true \
+--set ingressController.admissionWebhook.enabled=false
+```
+
+#### Detaching the broken KongPlugin
+
+As this mode of operation leverages the last valid Kubernetes objects' cache state, we need to make sure that
+we begin with a clean slate, allowing {{site.kic_product_name}} to store it. 
+Let's detach the broken `KongPlugin` from `route-b` so we get an entirely valid configuration:
+
+```bash
+kubectl annotate --overwrite httproute route-b konghq.com/plugins=
```

Comment:
```suggestion
kubectl annotate --overwrite httproute route-b konghq.com/plugins-
```

I believe we want to remove the annotation rather than set it to empty string.